src/deploy.sh

```diff
@@ -479,6 +487,29 @@ function modules_install_to()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function will copy and install the compiled dtb files
+# @target Where should the dtb files be installed (1-vm, 2-local or 3-remote)
+function dtb_install()
+{
+  local target="$1"
+  local architecture="${configurations[arch]:-'arm'}"
+
+  case "$target" in
+    1 | 3) # VM_TARGET or REMOTE_TARGET
+      cmd_remotely "mkdir -p $REMOTE_KW_DEPLOY/dts/overlays"
+      #copy dtb files to remote
+      cp_host2remote "arch/$architecture/boot/dts/*.dtb" "$REMOTE_KW_DEPLOY/dts"
+      cp_host2remote "arch/$architecture/boot/dts/overlays/*.dtb*" "$REMOTE_KW_DEPLOY/dts/overlays"
```

Comment:
My old (2019) raspbian installation doesn't have a dts directory. dtb files lay at the file system root dir (a.k.a. /) along with the kernel image and other configuration files. Yet talking about legacy raspberry systems, the overlays lay under "/overlays". I think we shouldn't assume "dts" to be part of either dtb or overlays path. Otherwise, add a note to warn users with older installs.

Reply:
@marceloschmitt1 I believe this may have been a mistake on my part, we create the dts folder on the remote only to better organize and decouple everything that needs to be installed, the idea is to rsync this dts folder to the boot folder. 
Rsyncinc to boot/dts instead of boot/ was a mistake, thank you for catching this!

Reply:
Oh, on may shallow review I missed you are creating the dts dir with `cmd_remotely`. I see your point, though I'm unsure if creating such directory structure worth it. I think it would be enough to `rsync` dtb files to / and dt-overlay files to /overlays directly. Of course, I'm not aware of how is your connection with the remote system so there might be technical issues I'm not seeing here.